app/src/layouts/appLayout/projectSidebar/organizationsBlock/organizationsBlock.jsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { useSelector } from 'react-redux';
+import { organizationNameSelector } from 'controllers/project';
+import styles from './organizationsBlock.scss';
+
+const cx = classNames.bind(styles);
+
+export const OrganizationsBlock = ({ openNavbar, openPopover }) => {
+  const organizationName = useSelector(organizationNameSelector);
+  const title = `${organizationName[0]}${organizationName[organizationName.length - 1]}`;
+
+  const onOpenNavbar = () => {
+    openPopover();
+    openNavbar();
+  };
+
+  return (
+    <button className={cx('organization-block')} tabIndex={0} onClick={onOpenNavbar}>
```

Comment:
Does the `tabIndex={0}` needed here?
I assume it should be 0 by default (see https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex#:~:text=Some%20focusable%20HTML%20elements%20have%20a%20default%20tabindex%20value%20of%200).